src/components/app/TabBar.js

```diff
@@ -43,9 +49,23 @@ class TabBar extends React.PureComponent<Props> {
               })}
               key={tabSlug}
               data-name={tabSlug}
-              onMouseDown={this._mouseDownListener}
+              onClick={this._onClickListener}
             >
-              {tabsWithTitle[tabSlug]}
+              {/* adding a button for better keyboard navigation  
+            (allows tabbing between the different panels)
+            ARIA attributes for better accessibility and usability
+            (arrow navigation between the tabs) */}
+              <button
+                className="tapBarTabButton"
+                type="button"
+                id={tabSlug}
```

Comment:
I think we need slightly more specific ids, so that we don't have possible conflicts later. What about:
```js
 {`${tabSlug}TabButton`}
```

Reply:
@gregtatum what's your advice for these tabslugs that aren't camelcased, eg `flame-graph`? We'd have `flame-graphTabButton` which is quite unpretty. Would you 1. don't care, 2. camel case them, 3. kebab case the whole thing? I don't have a strong opinion, so I'd choose 1 or 3 myself.

Reply:
What about:

`.tapBarTabButton.tab-slug-flame-graph`

That way `tapBarTabButton` would be the component class name, and then `tab-slug-flame-graph` can be a global modifier?

Reply:
Also, `tapBar` should probably be `tabBar`?

Reply:
@gregtatum the id here is not for styling, it's to connect the tab with its tabpanel (like label and id in form inputs). Don't know if that makes a difference. 
I kind of like flame-graph-tab-button, calltree-tab-button, what do you think?

Reply:
Let's do this